test/plugin/base_test.rb

```diff
@@ -2427,7 +2575,11 @@ def verify_field_key(log_entry_field, default_key, custom_key,
         d.emit(input[:emitted_log])
         d.run
       end
-      verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
+      verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload', false) do |entry|
+        if log_entry_field == 'labels'
+          input[:expected_field_value] = COMPUTE_PARAMS[:labels].merge(
```

Comment:
Can't you now do this unconditionally in `test_log_entry_labels_field`? If this is because of the hard-coded `nil` in the inputs, you could instead add an `expected_original_value` parameter that defaults to `nil`.

Reply:
Done